setup.py

```diff
@@ -19,7 +20,10 @@
     vertica.vertica_python = sqla_vertica_python.vertica_python:VerticaDialect
     """,
     install_requires=[
-        'vertica_python[namedparams]'
+        'vertica_python'
```

Comment:
This is the first functional change - fixes https://github.com/LocusEnergy/sqlalchemy-vertica-python/issues/36